assets/vendor/qtip2/jquery.qtip.js

```diff
@@ -1924,7 +1924,7 @@ QTIP.defaults = {
 			mouse: TRUE,
 			scroll: TRUE,
 			resize: TRUE,
-			method: 'flipinvert flipinvert'
+			method: 'shift shift'
```

Comment:
Is changing the defaults in the vendored code necessary? Why not add to `app.coffee` as done with viewport?

Reply:
I reverted my last commit and did the changes only on `app.coffe`